src/modules/simulation/gz_bridge/GZMixingInterfaceServo.hpp

```diff
@@ -66,11 +67,73 @@ class GZMixingInterfaceServo : public OutputModuleInterface
 	friend class GZBridge;
 
 	void Run() override;
+	int get_active_output_count();
+	float get_tilt_angle_max(const size_t index);
+	float get_tilt_angle_min(const size_t index);
+	float get_servo_angle_max(const size_t index);
+	float get_servo_angle_min(const size_t index);
+
+	/**
+	 * @brief Checks if the servo index is configured to tiltrotor.
+	 * Tiltrotors should be ordered after the control surfaces.
+	 * @param index: int
+	 * @return True if the index is as expected, otherwise False.
+	 */
+	bool is_tiltrotor(const int index);
+
+	/**
+	 * @brief Get maximum configured angle of servo.
+	 * Takes the maximum value from CA_SV_TL{n}_MAXA or SIM_GZ_SV_MAXA{n} param
+	 * dependent on input index.
+	 * @param index: servo index
+	 * @return angle_max [rad]
+	 */
+	double get_angle_max_wrapper(const int index);
+
+	/**
+	 * @brief Get minimum configured angle of servo.
+	 * Takes the minimum value from CA_SV_TL{n}_MINA or SIM_GZ_SV_MINA{n} param
+	 * dependent on input index.
+	 * @param index: servo index
+	 * @return angle_min [rad]
+	 */
+	double get_angle_min_wrapper(const int index);
 
 	gz::transport::Node &_node;
 	pthread_mutex_t &_node_mutex;
 
 	MixingOutput _mixing_output{"SIM_GZ_SV", MAX_ACTUATORS, *this, MixingOutput::SchedulingPolicy::Auto, false, false};
 
 	std::vector<gz::transport::Node::Publisher> _servos_pub;
+	std::vector<double> _angle_min_rad;
+	std::vector<double> _angular_range_rad;
+
+	DEFINE_PARAMETERS(
+		(ParamFloat<px4::params::CA_SV_TL0_MAXA>) _tilt_max_angle_0,
+		(ParamFloat<px4::params::CA_SV_TL0_MINA>) _tilt_min_angle_0,
+		(ParamFloat<px4::params::CA_SV_TL1_MAXA>) _tilt_max_angle_1,
+		(ParamFloat<px4::params::CA_SV_TL1_MINA>) _tilt_min_angle_1,
+		(ParamFloat<px4::params::CA_SV_TL2_MAXA>) _tilt_max_angle_2,
+		(ParamFloat<px4::params::CA_SV_TL2_MINA>) _tilt_min_angle_2,
+		(ParamFloat<px4::params::CA_SV_TL3_MAXA>) _tilt_max_angle_3,
+		(ParamFloat<px4::params::CA_SV_TL3_MINA>) _tilt_min_angle_3,
```

Comment:
Why do you need these parameters here? If the user have set them consistently, this should not need to be here

Reply:
Removed it

Reply:
and thanks a lot for the reviews!